Player.java

```diff
@@ -1,37 +1,24 @@
-import java.util.*;
-import java.io.*;
-
-public class Player{
-   private String name;
-   private int space;
+public class Player
+{
+   private final String name;
+   private int currentSpace;
    
-   public Player(String n){
-      space = 0;
-      name = n;
-   }
+   public Player(String n)
+   { currentSpace = 0; name = n; }
    
    public void setSpace(int s){
-      space = s;
+      currentSpace = s;
    }
       
    public String getName(){
       return name;
    }
    
-   public int getSpace(){
-      return space;
-   }
+   public int getSpace() { return currentSpace; }
    
-   public String toString(String name, int space){
-      return name + " is on space " + space;
-   }
+   public String toString()

```

Comment:
1. A requirement 2 implement this method was done kind of on purpose (tho a non-explained 1)
2. A convention 4 this method implementation is that it doesn't use any parameters. Besides, what u were passing in is readily available from inside the instance the method was being executed on.